tests/unit/test_generate.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+
+import flexmock
+import pytest
+
+
+from apigentools.commands.generate import GenerateCommand
+from apigentools.config import Config, LanguageConfig
+from apigentools.utils import run_command
+
+FIXTURE_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "fixtures",)
+
+
+# pull_repository
```

Comment:
I would add it as module docstring or completely remove it.

Reply:
I'm leaving in the FIXTURE_DIR as a module constant, since I read things from the fixtures directory. I'm not sure what you meant by "add it as a module docstring"?